utils/utils.js

```diff
@@ -45,9 +45,32 @@ const parseValidators = (validators) => {
   return errors;
 };
 
+
+/**
+ * Merges two JSON objects together, giving priority to the first object.
+ *
+ * @param {Object} obj1 - The first JSON object to be merged.
+ * @param {Object} obj2 - The second JSON object to be merged.
+ * @returns {Object} - The merged JSON object with properties from both obj1 and obj2
+ *
+ */
+function mergeWithPriority(obj1, obj2) {
```

Comment:
isn't this method equivalent to `Object.assign` ?
i.e, check if it can be implemented as:
```
function mergeWithPriority(obj1, obj2) {
  return Object.assign({}, obj2, obj1);
}
```